SofaKernel/modules/SofaCore/src/sofa/core/MultiMapping.h

```diff
@@ -213,7 +213,7 @@ class MultiMapping : public BaseMapping
             arg->logError("The 'input' data attribute is empty. It should contain a valid path "
                           "to one or more mechanical states of type '" + std::string(TIn::Name()) + "'.");
             return false;
-        } else if (!LinkFromModels::CheckPaths( input, context )) {
+        } else if (!PathResolver::CheckPaths(context, LinkFromModels::DestType::GetClass(), input)) {
```

Comment:
Look fine to me, except those lines which seems to not follow the topic of this PR

Reply:
You are right, I restored it. The PathResolver will come next in its own PR.